### PR TITLE
Add Review and Streamline Communications page

### PR DESCRIPTION
**Pull request by mark-pkgflow**
Card: [Create presentation page](https://trello.com/c/nPNePrB8)

image

I couldn’t find where this page should be linked from?

> You can access this page from the communications page in the address section or the explore resources page.

### CODE REVIEW

**Comment by mark-pkgflow** [comment]
[Create presentation page](https://trello.com/c/nPNePrB8/286-create-presentation-page)

**Comment by mark-pkgflow** [comment]
[Create presentation page](https://trello.com/c/nPNePrB8/286-create-presentation-page)

**Comment by mark-pkgflow** [comment]
[Create presentation page](https://trello.com/c/nPNePrB8/286-create-presentation-page)

**Comment by mark-pkgflow** [comment]
[Create presentation page](https://trello.com/c/nPNePrB8/286-create-presentation-page)

File: content/workload-reduction-toolkit/address-workload-issues/communications/review-and-streamline-communications.html.erb [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+          another member of staff to facilitate the session and lead on follow-up. You could also consider asking
+          somebody
+          independent to facilitate the workshop to free up time to take part.
+        </p>
+        <p>
+          Provide a supportive environment for open and honest professional dialogue.
+        </p>
+      </div>
+
+
```
Comment (AbigailMcP) [inline content/workload-reduction-toolkit/address-workload-issues/communications/review-and-streamline-communications.html.erb:87]: Extra lines

File: content/workload-reduction-toolkit/address-workload-issues/communications/review-and-streamline-communications.html.erb [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+      <div class="dfe-width-container govuk-grid-row">
+        <h2 class="govuk-heading-m">
+          Supporting resources
+        </h2>
+
+        <%= render('/partials/resource_card.*', title: "How we communicate" , href: "#{@base_url}/assets/files/Review school communications.odt" ,
+          body: "A template to help you explore how effective and efficient your communications are." , tag: "Example" ,
```
Comment (AbigailMcP) [inline content/workload-reduction-toolkit/address-workload-issues/communications/review-and-streamline-communications.html.erb:135]:
```suggestion
          body: "A template to help you explore how effective and efficient your communications are." , tag: "Template" ,
```

File: content/workload-reduction-toolkit/address-workload-issues/communications/review-and-streamline-communications.html.erb [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+          download_link: true) %>
+      </div>
+
+      <%= render('/partials/resource_card.*', title: "Impact graph" , href: "#{@base_url}/assets/files/Impact graph.odt" ,
+        body: "A template to help you prioritise what to address first." , tag: "Example", download_link: true ) %>
```
Comment (AbigailMcP) [inline content/workload-reduction-toolkit/address-workload-issues/communications/review-and-streamline-communications.html.erb:140]:
```suggestion
        body: "A template to help you prioritise what to address first." , tag: "Template", download_link: true ) %>
```

**Review by AbigailMcP — COMMENTED** [review]
Nice!

Just needs to be linked from the "Communications presentation" in `content/explore-all-resources/communications.html.erb` and `content/workload-reduction-toolkit/address-workload-issues/communications.html.erb`

File: content/workload-reduction-toolkit/address-workload-issues/communications.html.erb [hunk 4]
```diff
@@ -50,7 +50,7 @@ title: Communications
         <ul class="resource-card-group">
           <%= render('/partials/resource_card.*',
             title: "Communications presentation",
```
Comment (claire-hughez) [inline content/workload-reduction-toolkit/address-workload-issues/communications.html.erb:52]:
```suggestion
            title: "Review and streamline how we communicate",
```

File: content/workload-reduction-toolkit/address-workload-issues/communications/review-and-streamline-communications.html.erb [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+
+      <div class="dfe-width-container govuk-grid-row">
+        <h2 class="govuk-heading-m">
+          How to run with your staff
+        </h2>
+
+        <p>
+          You can run the exercises in a staff meeting or they can be completed individually or by teams ahead of
+          discussion in a
+          staff meeting.
```
Comment (claire-hughez) [inline content/workload-reduction-toolkit/address-workload-issues/communications/review-and-streamline-communications.html.erb:60]:
```suggestion
```

File: content/workload-reduction-toolkit/address-workload-issues/communications/review-and-streamline-communications.html.erb [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+      </div>
+
+      <div class="dfe-width-container govuk-grid-row">
+        <h2 class="govuk-heading-m">
+          How to run with your staff
+        </h2>
+
+        <p>
+          You can run the exercises in a staff meeting or they can be completed individually or by teams ahead of
+          discussion in a
```
Comment (claire-hughez) [inline content/workload-reduction-toolkit/address-workload-issues/communications/review-and-streamline-communications.html.erb:59]:
```suggestion
          the meeting.
```

File: content/workload-reduction-toolkit/address-workload-issues/communications/review-and-streamline-communications.html.erb [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+      </div>
+
+      <div class="dfe-width-container govuk-grid-row">
+        <h2 class="govuk-heading-m">
+          Supporting resources
+        </h2>
+
+        <ul class="resource-card-group">
+          <%= render('/partials/resource_card.*', title: "How we communicate" , href: "#{@base_url}/assets/files/Review school communications.odt" ,
+            body: "A template to help you explore how effective and efficient your communications are." , tag: "Example" ,
```
Comment (claire-hughez) [inline content/workload-reduction-toolkit/address-workload-issues/communications/review-and-streamline-communications.html.erb:133]:
```suggestion
            body: "A template to help you explore how effective and efficient your communications are." , tag: "Template" ,
```

File: content/workload-reduction-toolkit/address-workload-issues/communications/review-and-streamline-communications.html.erb [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+            download_link: true) %>
+            
+          <%= render('/partials/resource_card.*', title: "Impact graph" , href: "#{@base_url}/assets/files/Impact graph.odt" ,
+            body: "A template to help you prioritise what to address first." , tag: "Example", download_link: true ) %>
```
Comment (claire-hughez) [inline content/workload-reduction-toolkit/address-workload-issues/communications/review-and-streamline-communications.html.erb:137]:
```suggestion
            body: "A template to help you prioritise what to address first." , tag: "Template", download_link: true ) %>
```

File: content/workload-reduction-toolkit/address-workload-issues/communications/review-and-streamline-communications.html.erb [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+        <div class="dfe-width-container govuk-grid-row">
+          <p>
+            Use this presentation in a staff meeting or INSET day to review and streamline your internal and external
+            communication processes.
+          </p>
+          <p>
+            The presentation includes:
+          <ul>
+            <li>background and context about managing communications</li>
+            <li>a discussion to reflect on why we communicate and whether the processes should be updated, streamlined
```
Comment (claire-hughez) [inline content/workload-reduction-toolkit/address-workload-issues/communications/review-and-streamline-communications.html.erb:22]: could we separate this into 2 bullet points please:
<li>a reflection on why we communicate</li>
<li>a discussion on whether current communication processes should be updated, streamlined or stopped altogether</li>

**Review by claire-hughez — COMMENTED** [review]
Thanks Mark! Looks great. I wasn't sure how to make suggestions over multiple lines, so one is a comment.

I need to double check the location of the box to download with the design folks and will get back to you ASAP

**Comment by lauren-oseni-2** [comment]
Looks great - only comment from me is to adjust the spacing between the elements like so: 
Image 14-02-2024 at 16 01

**Comment by mark-pkgflow** [comment]
> Looks great - only comment from me is to adjust the spacing between the elements like so: Image 14-02-2024 at 16 01

Thanks @lauren-oseni-2 

That should be fixed now in case you want to check again.

**Comment by lauren-oseni-2** [comment]
Just one last spacing adjustment @mark-pkgflow! For the H1 + resource type tag

Image 15-02-2024 at 10 32

**Comment by mark-pkgflow** [comment]
> Just one last spacing adjustment @mark-pkgflow! For the H1 + resource type tag
> 
> Image 15-02-2024 at 10 32

@lauren-oseni-2 
How is this?

image

**Pull request merged**